ci-operator/step-registry/ipi/conf/vsphere/vips/ipi-conf-vsphere-vips-commands.sh

```diff
@@ -1,5 +1,16 @@
 #!/bin/bash
 
+grab_vip_seg() {
+  third_octet=$1
+  fourth_octet=$2
+  echo "192.168.${third_octet}.${fourth_octet}" >>"${SHARED_DIR}"/vips.txt
+}
```

Comment:
[OPNET-340](https://issues.redhat.com/browse/OPNET-340).  We always need an IPv6 VIP available for Dual-Stack conversion.

Reply:
Regardless of ipv6, there are no 192 subnets anymore. Not quite sure the intent here.